datafusion/proto/gen/Cargo.toml

```diff
@@ -35,4 +35,4 @@ workspace = true
 [dependencies]
 # Pin these dependencies so that the generated output is deterministic
 pbjson-build = "=0.7.0"
-prost-build = "=0.13.1"
```

Comment:
I needed to update prost-build too otherwise the CI was failing like this: https://github.com/apache/datafusion/actions/runs/10880552896/job/30187614122?pr=12483